contracts/UBI.sol

```diff
@@ -80,6 +87,32 @@ contract UBI is Initializable {
 
   /// @dev Timestamp since human started accruing.
   mapping(address => uint256) public accruedSince;
+  
+  /// @dev Timestamp of PoH activation, used as internal PoH Marker
+  mapping(address => bool) public activated;
+  
+  /// @dev Number of active Humans accruing UBI, Updated after adding / revoking PoH
+  uint256 public activeVerifiedHumans;
+
+  /// @dev Timestamp of last M0 update
+  uint256 public lastSupplyUpdate; // same as accruedSince but for M0 supply
+
+  /**
+  * @dev stream mapping
+  * stream[0][addr] = total incoming (excluding accruedPerSecond)
+  * stream[addr][0] = total outgoing
```

Comment:
Why not declare additional variables with more meaningful names (e.g. `totalIncoming` and `totalOutgoing`)? Would be easier to reason about below.

Reply:
`inflow[addr]` & `outflow[addr]` was merged down to `stream[src][dst]`.

Reply:
I'd highly recommend declaring mapping `totalIncoming[addr]` for `stream[0][addr]` , `totalOutgoing[addr]` for `stream[addr][0]`, and `counter[addr]` for `stream[this][addr]`. It is much more readable and I tested it locally and confirmed that it even performs slightly better because there are fewer map lookups this way. If there was some upstream discussion somewhere please link me, I'm just looking at the diff.

Reply:
It's readability vs. gas cost vs. proxy [storage collisions between versions](https://docs.openzeppelin.com/upgrades-plugins/1.x/proxies#storage-collisions-between-implementation-versions). 

It's ok to use 3+ slots for that NOW but sooner or later we'll upgrade again to use `struct` and time/value stream logic instead of ERC20 inspired ones.. so we'll get 3+ old storage slots hanging in all future codes that can't be easily reused for something else.